app/assets/javascripts/components/PipelineSampleReads.jsx

```diff
@@ -128,6 +115,29 @@ class PipelineSampleReads extends React.Component {
                           </table>
                         </div>
                       </div>
+                      <div className="row">
+                        <div className="col s12">
+                          <table>
+                            <tbody>
+                            <tr>
+                              <td className="notes">Notes</td>
+                              <td className="sample-notes">
+                                Subject 2
+                                <br/>
+                                248 6204  Taenia solium
+                                <br/>
+                                827 60517 Taenia asiatica
+                                <br/>
+                                -- Ignored pairs: 2787
+                                <br/>
+                                -- Retained pairs: 569 (16.95 percent of total non-host sequence)
+                                <br/>
+                              </td>
+                            </tr>
+                            </tbody>
+                          </table>
+                        </div>
+                      </div>
```

Comment:
When we have hardcoded test data like this, is there anything on the page to indicate this is fake data?